Firebase/Core/Private/FIRTimestamp+Internal.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2018 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "FIRTimestamp.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface FIRTimestamp ()
+
+/**
+ * Creates a new timestamp.
+ *
+ * @param seconds the number of seconds since epoch.
+ * @param nanoseconds the number of nanoseconds after the seconds.
+ */
+- (instancetype)initWithSeconds:(int64_t)seconds
```

Comment:
Since this essentially a POD type there's no harm in making this constructor a public one.

Reply:
Moved it to the public header.

I also moved `timestamp` factory function that returns a current timestamp to the public header, for consistency with Android (since we discussed today that `Timestamp#now' should exist on Android).

Let me know if you think `compare` should be moved to public as well. I'm less inclined towards `ISOString` - though it seems pretty cool, it doesn't currently exist on Android, so for consistency it would have to be implemented there as well.

Reply:
Made `compare` public.